internal/mcpserver/handler_test.go

```
@@ -0,0 +1,204 @@
+		}
+	}
+	for name, seen := range want {
+		if !seen {
+			t.Fatalf("missing tool %q", name)
+		}
+	}
+}
+
+func TestUnsupportedFutureProtocolFallsBack(t *testing.T) {
+	h, store := newTestHandler(t)
+	defer store.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `store.Close` is not checked

(errcheck)